packages/b2c-dx-mcp/package.json

```
@@ -64,8 +64,7 @@
     "lint": "eslint",
     "format": "prettier --write src",
     "format:check": "prettier --check src",
-    "preinspect": "pnpm run build",
-    "inspect": "mcp-inspector node bin/run.js --toolsets all --allow-non-ga-tools",
+    "inspect": "mcp-inspector node bin/dev.js --toolsets all --allow-non-ga-tools",
```

Here's what I was going to add in my PR. Giving us the option to run the inspector on the *built* project AND the *dev* project:



```suggestion
    "preinspect": "pnpm run build",
    "inspect": "mcp-inspector node bin/run.js --toolsets all --allow-non-ga-tools",
    "inspect:dev": "mcp-inspector node --conditions development bin/dev.js --toolsets all --allow-non-ga-tools",
```